sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/ConfigPropertiesTest.java

```diff
@@ -216,4 +218,31 @@ void durationUnitParsing() {
                 .getDuration("duration"))
         .isEqualTo(Duration.ofMillis(8));
   }
+
+  @Test
+  void defaultMethods() {
+    ConfigProperties properties = DefaultConfigProperties.get(emptyMap());
+    assertThat(properties.getBoolean("foo", true)).isTrue();
+    assertThat(properties.getString("foo", "bar")).isEqualTo("bar");
+    assertThat(properties.getDouble("foo", 65.535)).isEqualTo(65.535);
+    assertThat(properties.getInt("foo", 21)).isEqualTo(21);
+    assertThat(properties.getLong("foo", 123L)).isEqualTo(123L);
+    assertThat(properties.getDuration("foo", Duration.ofDays(13))).isEqualTo(Duration.ofDays(13));
+  }
+
+  @Test
+  void defaultCollectionTypes() {
+    ConfigProperties properties = DefaultConfigProperties.get(emptyMap());
+    assertThat(properties.getList("foo", Arrays.asList("1", "2", "3")))
+        .containsExactly("1", "2", "3");
+    assertThat(properties.getList("foo")).isEmpty();
+    Map<String, String> defaultMap = new HashMap<>();
+    defaultMap.put("one", "1");
+    defaultMap.put("two", "2");
+    Map<String, String> expected = new HashMap<>();
+    expected.put("one", "1");
+    expected.put("two", "2");
+    assertThat(properties.getMap("foo", defaultMap)).containsExactlyEntriesOf(expected);
```

Comment:
```suggestion
    assertThat(properties.getMap("foo", defaultMap)).containsExactly(
      entry("one", "1"), entry("two", "2"));
```

Alternatively `.containsExactlyEntriesOf(defaultMap)` seems pretty readable too